src/pybind/mgr/dashboard/frontend/src/app/shared/components/notifications-sidebar/notifications-sidebar.component.ts

```diff
@@ -164,4 +166,14 @@ export class NotificationsSidebarComponent implements OnInit, OnDestroy {
   trackByFn(index: number) {
     return index;
   }
+
+  silence(data: any) {
+    this.router.navigate(['/monitoring/silences/create'], {
+      state: {
+        title: data['title'],
+        matcher: 'alertname',
+        type: data['type']
+      }
+    });
+  }
```

Comment:
This will change to a new page, but if click the 🔕  icon in a pop-up notification I'd expect to stay in the page I was. What about calling the service in the background, instead of redirecting to a new page?

Reply:
Makes sense, I'l try that

Reply:
Done